### PR TITLE
Fix list wrap on second line

### LINKED ISSUE #4210: Lists don't wrap on second line
Right now we have this:

issue-listwraps

We can potentially fix with:

`list-style-position: outside;`

Maybe some margin/padding may also be needed.

### PR DESCRIPTION
**Pull request by Rahmon**
## Description
Fix list wrap on second line. Fixes #4210 

## How Has This Been Tested?
This has been tested with "npm test", "npm run test-e2e" and manually on Chrome and Firefox.

## Types of changes
Bug fix

## Checklist:
- [X] My code is tested.
- [X] My code follows the WordPress code style.
- [X] My code follows has proper inline documentation. #

### CODE REVIEW

**Comment by noisysocks** [comment]
Thanks for looking into this @Rahmon. The code looks good to me and this doesn't seem to cause any regressions.

Here's how it looks:

screen shot 2018-01-03 at 12 05 19

cc. @karmatosed

File: blocks/library/list/editor.scss [hunk 1]
```diff
@@ -2,6 +2,5 @@
 .blocks-list .blocks-editable__tinymce ul,
 .blocks-list .blocks-editable__tinymce ol {
 	padding-left: 1.3em;
-	list-style-position: inside;
-	margin-left: 0;
+	margin-left: 1.3em;
```
Comment (mtias) [inline blocks/library/list/editor.scss]: I believe the "inside" was added for some toolbar issues. cc @gziolo

Comment (mtias) [reply blocks/library/list/editor.scss]: It was added here: https://github.com/WordPress/gutenberg/commit/1c53b02039be3d318350dfa08badfe90fd487af8

**Comment by jorgefilipecosta** [comment]
Hi @Rahmon, thank you for your contribution 👍  The inside was added as a fix to the bugs reported in https://github.com/WordPress/gutenberg/issues/3727, https://github.com/WordPress/gutenberg/issues/3707. This change does not create exactly the same bug, but affects the bullet point position creating a new undesirable behavior:
image
Can be replicated with the following post content https://gist.github.com/jorgefilipecosta/47b6fbac58b6ce66b8a1ffc2fdf02427. This strange behavior is much harder to replicate than the original bugs and the dimensions need to be really specific. I don't see any reason for this bugs to be happening, it looks like a random browser rendering problem and seems to just happen in chrome.

**Comment by jorgefilipecosta** [comment]
I did some debugging and it looks like I found a workaround for the new spacing bug not involving list-style-position. So I think this problem is not a blocker and the changes can be merged and I will create a parallel PR that addresses this spacing bullet position problem.

**Comment by Rahmon** [comment]
@jorgefilipecosta I saw that you did the last change in editor.scss file so I verified the bugs reported in #3727 and #3707 before submit this PR.. xD

**Pull request merged**